cache/cache.cpp

```diff
@@ -281,10 +603,10 @@ int dnet_cmd_cache_lookup(struct dnet_backend_io *backend, struct dnet_net_state
 		return -ENOTSUP;
 	}
 
-	cache_manager *cache = (cache_manager *)backend->cache;
+	auto cache = reinterpret_cast<cache_manager *>(backend->cache);
```

Comment:
It gives no guarantees here in comparison with reinterpret_cast.
C-style raw pointer should be converted using reinterpret_cast.